.travis.yml

```diff
@@ -33,19 +33,26 @@ after_failure:
 # Integration tests need to run first to reset the PR build status to pending
 stages:
   - 'Integration tests'
+  - 'Benchmarking tests'
   - 'Test'
 
 jobs:
   include:
-    - stage: 'Integration tests'
+    - &integrationtest
+      stage: 'Integration tests'
       merge_mode: replace
       env: SDK=java
       cache: false
       language: minimal
       before_install: skip
       install: skip
       before_script:
-        - mkdir $HOME/travisci-tools && pushd $HOME/travisci-tools && git init && git pull https://$CI_USER_TOKEN@github.com/optimizely/travisci-tools.git && popd
+        - mkdir $HOME/travisci-tools && pushd $HOME/travisci-tools && git init && git pull https://$CI_USER_TOKEN@github.com/optimizely/travisci-tools.git sohail/refactorscript && popd
```

Comment:
Please remove branch, code is merged in master.

Reply:
Ok